src/components/stopwatch/StopWatchController.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <FreeRTOS.h>
+#include <optional>
+#include <timers.h>
+
+namespace Pinetime {
+  namespace System {
+    class SystemTask;
+  }
+  namespace Controllers {
+
+    enum class StopWatchStates { Cleared, Running, Paused };
+
+    struct LapInfo {
+      int count = 0;       // Used to label the lap
+      TickType_t time = 0; // Delta time from beginning of stopwatch
+    };
+
+    constexpr int lapCapacity = 2;
```

Comment:
I think this needs to go inside StopWatchController as it should be namespaced to it? You could have it as a private member with static constexpr